Assets/HoloToolkit/UX/Scripts/Collections/ObjectCollection.cs

```diff
@@ -395,6 +350,65 @@ public void UpdateCollection()
             }
         }
 
+        /// <summary>
+        /// Update the facing of a node given the nodes new position for facing orign with node and orientation type
+        /// </summary>
+        /// <param name="node"></param>
+        /// <param name="orientType"></param>
+        /// <param name="newPos"></param>
+        private void UpdateNodeFacing(CollectionNode node, OrientTypeEnum orientType, Vector3 newPos = new Vector3())
```

Comment:
instead of creating a new vector3, let's use Vector3.zero

Reply:
Vector3.zero isn't a compile-time constant, so it can't be used as a default value. How about:

UpdateNodeFacing(CollectionNode node, OrientTypeEnum orientType, __Vector3 newPos = default(Vector3)__)

Reply:
Didn't think it would be, but it was worth a shot.